eq-author/src/components/QuestionConfirmationRoute/Editor.js

```diff
@@ -48,6 +48,7 @@ export class UnwrappedEditor extends React.Component {
       <Wrapper>
         <RichTextEditor
           id="confirmation-title"
```

Comment:
You can remove this id as its not used anymore.

Reply:
`id`'s a required prop in the RTE, so I'll have a look at if that still needs to be the case in the refactor I'll do in the next pr.